iOS-sample-app/Showcases/Screen Views/DefaultScreenViewController.swift

```diff
@@ -0,0 +1,21 @@
+
+import UIKit
+import ContentSquare
+
+class DefaultScreenViewController: UIViewController {
+
+    // In most cases, the simplest place to track screen views is in the controller's
+    // viewDidAppear.
+    //
+    // Avoid putting screen views in viewWillAppear. It is advised to avoid puting non critical
+    // work in viewWillAppear, since you don't want to use the main thread unnecessarily while
+    // a view is in the process of being displayed.
+    //
+    // Also, avoid using viewDidLoad, since it won't always be called. For instance, it won't
+    // be called in cases of back navigation in a navigation stack.
```

Comment:
I think you can be more direct and strict in the directions here. viewDidAppear is not the simplest, but the right place to track a screenview. It is the function that is called when we know for a fact that the screen is visible, everytime it is. Not sure you have to get into much details into why viewDidLoad and viewWillAppear are not suited, but you could keep that if you want I guess. Just make sure you are more forceful in your language, so as to leave no doubt in the client's mind.

Reply:
Ok, changed the wording. It also made the statements more lightweight. I'd rather keep the notes on the methods not to use since people could very well think that it's equivalent.